api/pluginAPI-V0.yml

```diff
@@ -105,6 +105,26 @@ paths:
           description: Error retrieving logs
           schema:
             $ref: "#/definitions/Error"
+
+  /api/autoRestart:
```

Comment:
It would be wise to anticipate the future needs here.
You should create a more generic endpoint.
something like "config" that will serve for autorestart option, but also for futur options we will most probably need